partiql-eval/src/main/java/org/partiql/eval/Environment.java

```diff
@@ -73,10 +73,10 @@ public String toString() {
             sb.append("$i: $row");
             sb.append("---------------------");
         }
-//        if (scope.isEmpty()) {
-//            appendLine("empty")
-//            appendLine("---------------------")
-//        }
-        return "stack";
+        if (stack.length == 0) {
+            sb.append("empty\n");
+            sb.append("---------------------\n");
+        }
+        return sb.toString();
```

Comment:
this was to fix pretty-printing the stack on evaluator errors (shouldn't happen)